docs/Development.md

```diff
@@ -314,9 +314,9 @@ We are in the process of refactoring our helm charts such that they can be teste
 
 The following is a walkthrough of the process we follow to create and manage versioned releases of Pelorus.
 
-1. Update the default exporter tag version to the new tag you're about to make.  
```

Comment:
Two trailing spaces in markdown is significant. It puts a `<br>` instead of starting a new `<p>`. Can you put these back?